experiments/clean_csv.py

```
@@ -0,0 +1,270 @@
+
+def get_or_create_recipient(row):
+    """Get or create recipient record and return id."""
+    recipient_name = row.get('RCPNT_NML_EN_DESC', '')
+    recipient_class = row.get('RCPNT_CLS_EN_DESC', '')
+    city = row.get('CTY_EN_NM', '')
+    province = row.get('PROVTER_EN', '')
+    country = row.get('CNTRY_EN_NM', '')
+    
+    # Skip if empty
+    if not recipient_name:
+        raise "Missing recipient name"
```

Ruff (0.8.2)

Cannot raise a literal. Did you intend to return it or raise an Exception?

(B016)